core/mempool/src/pool.rs

```diff
@@ -195,18 +206,32 @@ impl PriorityPool {
                 *reduce_len += 1;
             }
         }
+
+        let timeout = if number > self.timeout_config {
+            timeout_gap.remove(&number.saturating_sub(self.timeout_config))
```

Comment:
now that `number` is larger than `timeout_config`,  why bother to use `saturating_sub`?  
why not `number - self.timeout_config`?